docs/intro.rst

```diff
@@ -0,0 +1,7 @@
+===============
+Getting started
+===============
+
+TODO: Write about how to include Resolwe in a Django project and explain
+settings parameters. Create an example Django project in the ``docs/example``
```

Comment:
Do we want to create another project and mantain it? We already have minimal project for running tests. Is it ok to add comments there?